scripts/lib/spawn-sync.js

```diff
@@ -0,0 +1,24 @@
+const processSpawnSync = require('child_process').spawnSync
```

Comment:
I'd go with `execa` for less stuff to maintain

Reply:
oh I never use that before, will check it out. Thanks !

Reply:
would you please check again @thymikee ?